modules/rules_java/metadata.json

```diff
@@ -1,13 +1,11 @@
 {
     "homepage": "https://github.com/bazelbuild/rules_java",
     "maintainers": [],
-    "repository": [
-        "github:bazelbuild/rules_java"
-    ],
```

Comment:
@comius You may also want to adjust your `.bcr/template.metadata.json` to include this `repository` field.